libraries/RC_Channel/RC_Channels.cpp

```diff
@@ -341,6 +347,49 @@ RC_Channel &RC_Channels::get_yaw_channel()
 #endif  // AP_RCMAPPER_ENABLED
 
 
+/*
+  check for pilot input on rudder stick for arming/disarming
+*/
+void RC_Channels::rudder_arm_disarm_check()
+{
+    const RC_Channel *channel = get_arming_channel();
+    if (channel == nullptr) {
+        return;
+    }
+
+    const auto control_in = abs(channel->get_control_in());
+
+    if (control_in == 0) {
+        have_seen_neutral_rudder = true;
+    }
+
+    if (control_in <= 4000) {
+        // not trying to (or no longer trying to) arm or disarm
+        rudder_arm_timer = 0;
+        return;
+    }
+
+    const uint32_t now = AP_HAL::millis();
+    if (rudder_arm_timer == 0) {
+        // first time we've seen the attempt
+        rudder_arm_timer = now;
+        return;
+    }
+
+    if (now - rudder_arm_timer < 3000) {
+        // not time yet....
+        return;
+    }
+
```

Comment:
I would have expected a `rudder_arm_timer = 0` here somewhere so you get one arm attempt per 3 seconds rather than continual attempts after 3 seconds.

Reply:
Fixed, thanks.  It was a problem.